### ISSUE TITLE
Bitstamp V2 async fails to authenticate

### ISSUE THREAD

**Issue by mandrikin** [ccxt/ccxt#6846]
- OS: N/A
- Programming Language version: Python 3.7
- CCXT version: 1.26.85
- Exchange: Bitstamp V2 async

When trying to call any of the Bitstamp V2 async endpoints which require authentication the following error is thrown:

```
Request: POST https://www.bitstamp.net/api/v2/balance/ {'X-Auth': 'BITSTAMP DELETED', 'X-Auth-Nonce': 'DELETED', 'X-Auth-Timestamp': 'DELETED', 'X-Auth-Version': 'v2', 'X-Auth-Signature': b'15d937c908998dd0d88adc7e575812b590b91f090631189eca7ff523fe68fff4', 'User-Agent': 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/62.0.3202.94 Safari/537.36', 'Accept-Encoding': 'gzip, deflate'} None
Traceback (most recent call last):
  File "~/lib/python3.7/site-packages/ccxt/async_support/bitstamp.py", line 516, in fetch_balance
    balance = await self.privatePostBalance(params)
  File "~/lib/python3.7/site-packages/ccxt/async_support/base/exchange.py", line 95, in fetch2
    return await self.fetch(request['url'], request['method'], request['headers'], request['body'])
  File "~/lib/python3.7/site-packages/ccxt/async_support/base/exchange.py", line 120, in fetch
    proxy=self.aiohttp_proxy) as response:
  File "~/lib/python3.7/site-packages/aiohttp/client.py", line 1012, in __aenter__
    self._resp = await self._coro
  File "~/lib/python3.7/site-packages/aiohttp/client.py", line 502, in _request
    resp = await req.send(conn)
  File "~/lib/python3.7/site-packages/aiohttp/client_reqrep.py", line 629, in send
    await writer.write_headers(status_line, self.headers)
  File "~/lib/python3.7/site-packages/aiohttp/http_writer.py", line 111, in write_headers
    buf = _serialize_headers(status_line, headers)
  File "aiohttp/_http_writer.pyx", line 138, in aiohttp._http_writer._serialize_headers
  File "aiohttp/_http_writer.pyx", line 110, in aiohttp._http_writer.to_str
**TypeError: Cannot serialize non-str key b'15d937c908998dd0d88adc7e575812b590b91f090631189eca7ff523fe68fff4'**
```

I saw at least 2 other issues (like this one: https://github.com/ccxt/ccxt/issues/5229) with the aiohttps library not liking byte objects in the headers.
So it might be a good idea to sift through all async supported exchanges and fix all instances of this issue.

In this case changing 
```
headers['X-Auth-Signature'] = signature
```
to
```
headers['X-Auth-Signature'] = self.decode(signature)
```
fixes this particular issue.

But issuing another request throws this error:
```
ccxt.base.errors.ExchangeError: bitstamp {"status": "error", "reason": "Content-Type header should not be present", "code": "API0020"}
```

Here is the request:
```
Request: POST https://www.bitstamp.net/api/v2/balance/ {'X-Auth': 'BITSTAMP DELETED', 'X-Auth-Nonce': 'DELETED', 'X-Auth-Timestamp': 'DELETED', 'X-Auth-Version': 'v2', 'X-Auth-Signature': 'da7090000c730008df33c154eaff94c23b030689f930994cc1f7313d37cb5821', 'User-Agent': 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/62.0.3202.94 Safari/537.36', 'Accept-Encoding': 'gzip, deflate'} None

Response: POST https://www.bitstamp.net/api/v2/balance/ 403 <CIMultiDictProxy('Date': 'Thu, 23 Apr 2020 20:36:29 GMT', 'Content-Type': 'application/json', 'Transfer-Encoding': 'chunked', 'Connection': 'keep-alive', 'Server': 'Apache', 'X-XSS-Protection': '1; mode=block', 'Content-Language': 'en', 'Expires': 'Thu, 23 Apr 2020 20:36:29 GMT', 'Vary': 'Authorization,Accept-Language,Cookie', 'Content-Security-Policy-Report-Only': "default-src 'self' 'report-sample'; connect-src 'self' wss://ws.pusherapp.com wss://ws.bitstamp.net *.pusher.com *.trackjs.com *.google-analytics.com stats.g.doubleclick.net ; font-src 'self' data: assets.bitstamp.net fonts.gstatic.com www.google.com ; frame-ancestors 'self'; frame-src 'self' pixel-a.basis.net 8720977.fls.doubleclick.net *.ledgerwallet.com *.google.com www.googletagmanager.com pixel.sitescout.com ctpe.net ; img-src * data: blob:; report-uri /api/report-csp/; script-src 'self' 'unsafe-eval' assets.bitstamp.net js-agent.newrelic.com *.google-analytics.com *.pusher.com d3dy5gmtp8yhk7.cloudfront.net www.googleadservices.com www.googletagmanager.com www.gstatic.com www.recaptcha.net code.highcharts.com/stock/highstock.js bam.nr-data.net 'nonce-MjEyMjU0MDc5NTk0NTQwMzc2OTIxMzM1MDc1Njc3MDg2MzcyNTk5'; style-src 'self' 'unsafe-inline' assets.bitstamp.net fonts.googleapis.com", 'Cache-Control': 'max-age=0', 'X-Frame-Options': 'SAMEORIGIN', 'x-bts-trace-id': 'Root=1-5ea1fc4d-6a7d19c87df632cb901589fb', 'Strict-Transport-Security': 'max-age=63072000; includeSubDomains', 'Last-Modified': 'Thu, 23 Apr 2020 20:36:29 GMT', 'Access-Control-Allow-Origin': '*', 'Access-Control-Allow-Methods': 'POST, GET', 'Access-Control-Allow-Headers': 'x-requested-with, Content-Type, origin, accept, cache-control', 'X-CDN': 'Incapsula', 'X-Iinfo': '7-7425470-7423888 PNNy RT(1587674187761 895) q(0 0 0 0) r(0 0) U6')> {"status": "error", "reason": "Content-Type header should not be present", "code": "API0020"}
```
..which is strange because there is no Content-Type header set in the POST request.

**Comment by kroitor** [comment]
> ..which is strange because there is no Content-Type header set in the POST request.

Yes, it is strange – added a workaround for it.

This issue is fixed in version 1.26.90, the new build will arrive in 15 minutes. We will be happy if you  report whether the issue is fixed on your side or not after upgrading. Thx for your feedback!

**Comment by mandrikin** [comment]
Retested the issue in 1.26.90 and it is fixed.

Thanks for the quick response!
